lib/connection.c

```diff
@@ -670,10 +670,19 @@ json_copy_string(char **dest, cJSON *data, const char *item) {
     size_t blob_size;
     cJSON *blob = snowflake_cJSON_GetObjectItem(data, item);
     if (!blob) {
+        // We don't check the return status everywhere
+        // make sure that the value is set to NULL to enable
+        // NULL checks.
+        SF_FREE(*dest);
+        *dest = NULL;
```

Comment:
The SF_FREE macro already sets *dest to NULL after it frees the data, so this line is redundant